content/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

```diff
@@ -25,8 +25,8 @@
         </div>
         <div class="row">
             <div class="col-md-12">
-                <a class="btn btn-lg btn-success" href="/ko/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">
-                    모듈 2로 진행하기<span class="btn__next">›</span></a>
+              <a class="btn btn-lg btn-success" href="/ko/docs/tutorials/kubernetes-basics/" role="button">홈으로 이동<span class=""></span></a>
+                <a class="btn btn-lg btn-success" href="/ko/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">모듈 2로 진행하기 &gt<span class=""></span></a>
```

Comment:
```suggestion
                <a class="btn btn-lg btn-success" href="/ko/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">모듈 2로 진행하기 &gt;<span class=""></span></a>
```
대부분의 요새 브라우저에서는 정상적으로 나오지만, 사실 표준에 어긋난 코딩이라 올려봅니다.
(참고: https://dev.w3.org/html5/html-author/charref)

사실 요기만 아니라, **meta** 태그의 내용도 다 수정해야 합니다만, 이건 다른 PR로 처리해야 할 듯 싶네요.